content/home/examples/a-simple-component.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: A Simple Component
+title: ఎ సింపుల్ కాంపోనెంట్
 order: 0
 domid: hello-example
 ---
 
-React components implement a `render()` method that takes input data and returns what to display. This example uses an XML-like syntax called JSX. Input data that is passed into the component can be accessed by `render()` via `this.props`.
+రియాక్ట్ కంపోనెంట్లు ఇన్పుట్ డేటాను తీసుకువచ్చే ఒక `render()` పద్ధతిని అమలు చేస్తాయి మరియు దాన్ని ప్రదర్శించడానికి తిరిగి పంపుతుంది. ఈ ఉదాహరణ JSX గా పిలువబడే ఒక XML-వంటి సింటాక్స్ ను ఉపయోగిస్తుంది. కంపోనెంట్నికి పంపుతున్న ఇన్పుట్ డేటాను `this.props` ద్వారా `render()` ద్వారా ప్రాప్తి చేయవచ్చు.
```

Comment:
రియాక్ట్ కంపోనెంటు `render()` అనే ఒక మెథడ్ని వాడుతుంది, ఇది ఇన్పుట్ డేటా తీసుకుని ఏమి చూపించాలో దాన్ని ఇస్తుంది. ఈ ఉదాహరణ XML-లాగ కనిపించే JSX అనే సింటాక్స్ వాడుతోంది. ఈ కంపోనెంటుకు పంపించే ఇన్పుట్ డేటాను `this.props` ద్వారా `render()` మెథడ్ చూడగల్గుతుంది.